snorkel/classification/loss.py

```diff
@@ -0,0 +1,122 @@
+from typing import Callable, List, Mapping, Optional, Sequence, Tuple, Union
+
+import torch
+import torch.nn.functional as F
+
+Outputs = Mapping[str, List[torch.FloatTensor]]
+
+
+def cross_entropy_soft(
+    input: torch.FloatTensor,
```

Comment:
Yeah, I was torn on this. Originally did `inputs` instead to avoid the keyword, but ended up sticking with `input` to mirror PyTorch's `F.cross_entropy()` signature.